test/e2e/base/manifests.yaml

```diff
@@ -200,7 +200,7 @@ metadata:
   labels:
     app: infra-backend-v3
 spec:
-  replicas: 2
+  replicas: 4
```

Comment:
Do we have to increase the replicas?

Reply:
for testing a load-balancer, just 2 endpoints seems a little insufficient, would be nice if we can add more endpoints.

rethink about this setup, maybe we should just create a deployment with replicas=4 only for LB test case, not other cases. could save some resources this way.